drizzlepac/hlautils/analyze.py

```diff
@@ -43,6 +45,43 @@ class Messages(Enum):
 
     OK, WARN, NOPROC = 1, -1, -2
 
+
+def analyze_wrapper(input_file_list):
+    """
+    Thin wrapper for the analyze_data function to return a list of viable images.
+
+    Parameters
+    ==========
+    input_file_list: list
```

Comment:
```suggestion
    input_file_list : list
```
This simple change will allow Sphinx to render the parameter correctly.

Reply:
Fixed.